website/docs/r/cis_origin_auth.html.markdown

```diff
@@ -0,0 +1,74 @@
+
+## Example usage
+
+```terraform
+# Change MTLS setting of CIS instance
+
+resource "ibm_cis_mtls" "mtls_settings" {
+  cis_id                          = data.ibm_cis.cis.id
+  domain_id                       = data.ibm_cis_domain.cis_domain.domain_id
+  certificate                     = EOT<<
+                                  "-----BEGIN CERTIFICATE----- 
+                                  --------END CERTIFICATE-----"
+                                  EOT
+  private_key                     = <<EOT 
+                                  "-----  BEGIN RSA PRIVATE KEY    ----- 
+                                  -------   END RSA PRIVATE KEY    -----"
+                                  EOT
+  hostname                        = "abc.abc.abc.com"
+}
+```
+
+## Argument reference
+
+Review the argument references that you can specify for your resource. 
+
+- `cis_id`                  - (Required, String) The ID of the IBM Cloud Internet Services instance.
+- `domain_id`               - (Required, String) The ID of the domain to change cache settings.
+- `certificate`             - (Required, String) Content of certificate.
+- `private_key`             - (Required, String) Content of private key.
+- `hostname`                - (optional, String) Valid host names for host level origin auth processing.
+- `enabled`                 - (optional, Bool)   Default is true, it enables/disables the host and zone level origin auth setting .
+
+
+## Attribute reference
+In addition to all argument reference list, you can access the following attribute reference after your resource is created.
+
+- `auth_id`      - (Computed, String) The record ID. It is a combination of `<mtls_id>, <domain_id>,<cis_id>` attributes concatenated with `:`.
+- `updated_at`   - (Computed, String) Time stamp string when Certificate is modified'.
+- `expires_on`   - (Computed, String) Time stamp string when Cerftificate expires on'.
+- `cert_id`      - (Computed, String) Uploaded certificate ID.
+- `status`       - (Computed, String) Origin auth status enbled or not.
+
+
+## Import
+The `ibm_cis_orig_auth` resource can be imported using the ID. The ID is formed from the domain ID of the domain and the CRN concatenated  using a `:` character.
+
+The domain ID and CRN will be located on the overview page of the IBM Cloud Internet Services instance of the console domain heading, or by using the `ibmcloud cis` command line commands.
+
+- **Domain ID** is a 32 digit character string of the form: `9caf68812ae9b3f0377fdf986751a78f`
+
+- **CRN** is a 120 digit character string of the form: `crn:v1:bluemix:public:internet-svcs:global:a/4ea1882a2d3401ed1e459979941966ea:31fa970d-51d0-4b05-893e-251cba75a7b3::`
+
+**Syntax**
+
+```
+$ terraform import ibm_cis_orig_auth.origin_auth_settings <domain-id>:<crn>
```

Comment:
This import stmt is wrong its combination of more than 2 elements

Reply:
OK.